app/src/modules/content/components/comparison-header.vue

```diff
@@ -0,0 +1,302 @@
+}>();
+
+const userUpdatedName = computed(() => {
+	if (!props.userUpdated) return null;
+	return userName(props.userUpdated);
+});
+
+const deltaOptions = computed(() => {
+	if (!props.showDeltaDropdown || !props.comparisonData?.selectableDeltas) return [];
+
+	const deltas = props.comparisonData.selectableDeltas;
+	return deltas.map((delta: any) => {
+		if (delta.activity?.timestamp) {
+			const date = localizedFormat(new Date(delta.activity.timestamp), String(t('date-fns_date_short')));
+			const time = localizedFormat(new Date(delta.activity.timestamp), String(t('date-fns_time')));
+			const formattedDate = `${date} (${time})`;
+
+			let user = t('private_user');
+
+			if (typeof delta.activity.user === 'object') {
+				user = userName(delta.activity.user);
+			}
+
+			return {
+				text: `${formattedDate} - ${user}`,
+				value: delta.id,
+			};
+		}
+
+		return {
+			text: delta.name || delta.key || `Item ${delta.id}`,
+			value: delta.id,
+		};
+	});
+});
+
+const selectedDeltaId = computed(() => {
+	if (!props.showDeltaDropdown || !props.comparisonData) return null;
+
+	return props.comparisonData.initialSelectedDeltaId || null;
+});
+
+const selectedOption = computed(() => {
+	if (!selectedDeltaId.value) return null;
+	return deltaOptions.value.find((option: any) => option.value === selectedDeltaId.value);
+});
+</script>
+
+<template>
+	<div class="comparison-header">
+		<div class="header-content">
+			<div class="title-container">
+				<template v-if="loading">
+					<v-skeleton-loader type="text" class="title-skeleton" />
+				</template>
+				<template v-else>
+					<h3>{{ title }}</h3>
+					<v-chip v-if="showLatestChip" small class="latest-chip">{{ t('latest_version') }}</v-chip>
+				</template>
+			</div>
+		</div>
+		<div class="header-meta">
+			<template v-if="loading">
+				<div class="meta-skeleton">
+					<v-skeleton-loader type="text" class="date-skeleton" />
+				</div>
+			</template>
+			<template v-else>
+				<div v-if="showDeltaDropdown" class="delta-dropdown">
+					<v-menu show-arrow>
+						<template #activator="{ toggle }">
+							<div class="delta-selection" @click="toggle">
+								<div class="delta-content">
+									<div class="delta-date-time">{{ selectedOption?.text.split(' - ')[0] }}</div>
+									<div class="delta-user-info">{{ t('edited_by') }} {{ selectedOption?.text.split(' - ')[1] }}</div>
+								</div>
+								<v-icon name="expand_more" class="dropdown-icon" />
+							</div>
+						</template>
+						<v-list>
+							<v-list-item
+								v-for="option in deltaOptions"
+								:key="option.value"
+								:active="selectedDeltaId === option.value"
+								clickable
+								@click="emit('delta-change', option.value)"
+							>
+								<v-list-item-content>
+									<div class="delta-option-content">
+										<div class="delta-option-date-time">{{ option.text.split(' - ')[0] }}</div>
+										<div class="delta-option-user-info">{{ option.text.split(' - ')[1] }}</div>
```

Comment:
Let’s use the same representation as above.

```suggestion
										<div class="delta-option-user-info">{{ t('edited_by') }} {{ option.text.split(' - ')[1] }}</div>
```